packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -203,6 +203,14 @@ export default function({ types: t }) {
           path.ensureBlock();
         }
       },
+      CallExpression(path) {
+        const { node } = path;
+        if (node.callee.name === "_initializerWarningHelper") {
```

Comment:
🤔 is this good enough? is created via `generateUidIdentifier`, so the name could end up being different?

Reply:
Yeah shouldn't be hardcoding it, but It's probably fine in most cases.

```
function _initializerWarningHelper(descriptor, context) {
  throw new Error('Decorating class property failed. Please ensure that ' + 'transform-class-properties is enabled.');
}
```

Although looking at this, we might as well just change the error message here?

Reply:
hm good point, I had been thinking it would be ideal to get an error at compile time, but that probably gets the point across too

Reply:
Oh ok yeah that's true too, ideally we would just know the plugins you run and have a place to just throw immediately (plugin ordering PR should of helped with that)

Reply:
Meanwhile, I suppose one advantage of leaving it at runtime is that if someone is using spec mode class properties with legacy decorators (for example via the stage 1 preset), they won't encounter any issues unless they are actually attempting to decorate class properties.